dfsclient/src/main/java/org/hpccsystems/dfs/client/HpccRemoteFileReader.java

```diff
@@ -47,10 +47,33 @@ public HpccRemoteFileReader(DataPartition dp, FieldDef originalRD, IRecordBuilde
      *            the record defintion for the dataset
      * @param recBuilder
      *            the IRecordBuilder used to construct records
+     * @param connectTimeout 
+     *            the connectiont timeout in seconds
      * @param projectedRD
      *            the requested record format
      */
     public HpccRemoteFileReader(DataPartition dp, FieldDef originalRD, IRecordBuilder recBuilder, int connectTimeout) throws Exception
+    {
+        this(dp,originalRD,recBuilder,connectTimeout,-1);
```

Comment:
Can we create a Symbol to signify no limit, that way we can change the actual value behind the scenes if we need to